### PR TITLE
dynamically import dateformatter subclasses

### PR DESCRIPTION
**Pull request by rlskoeser**
Addresses #23  by adding a one-time method to import all local `DateFormatter` subclasses so they will be defined when `available_subclasses` is called.

### CODE REVIEW

File: tox.ini [hunk 1]
```diff
@@ -30,7 +31,9 @@ commands =
 [testenv:coverage]
 deps =
     pytest
+    pytest-ordering
     pytest-cov
+
```
Comment (rlskoeser) [inline tox.ini:36]: Can someone who knows tox tell me the right way to do this?

Seems like we shouldn't have to add dependencies in more than one place, but I couldn't get the tests passing without this.  Maybe the way I created the coverage command is part of the problem, but even so we're already specifying test deps in our `setup.cfg` — can we use that here somehow?

Comment (ColeDCrawford) [reply tox.ini:36]: Haven't used `tox` except here unfortunately. Looks like these are some instructions on how to configure `tox` to use `setup.cfg` settings: https://tox.wiki/en/latest/config.html#configuration-discovery. Found that from https://stackoverflow.com/questions/67098213/is-there-a-reason-to-have-both-tox-ini-and-setup-cfg-in-a-python-project

Comment (rlskoeser) [reply tox.ini:36]: Thanks @ColeDCrawford — I saw in passing that tox could be configured in `setup.cfg`, that sounds appealing to me but not sure the implications

**Comment by rlskoeser** [comment]
Ok, tox / setup.cfg stuff is resolved and checks are passing now. @ColeDCrawford or @jdamerow if one of you could review, I think we can merge (or I can address if you have questions/concerns)

**Review by jdamerow — APPROVED** [review]

**Pull request merged**